CONTRIBUTING.md

```diff
@@ -1,5 +1,74 @@
-# CONTRIBUTING
+# Contributing to Mistral AI Client for Dart
 
-## Development flow
+Thank you for considering contributing to [Your Project Name]! We appreciate your time and effort.
 
-## How to build
+## Table of Contents
+
+- [Contributing to Mistral AI Client for Dart](#contributing-to-mistral-ai-client-for-dart)
+  - [Table of Contents](#table-of-contents)
+  - [How Can I Contribute?](#how-can-i-contribute)
+    - [Reporting Bugs](#reporting-bugs)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+    - [Code Contributions](#code-contributions)
+    - [Documentation](#documentation)
+  - [Getting Started](#getting-started)
+    - [Setting Up the Development Environment](#setting-up-the-development-environment)
+  - [Style Guide](#style-guide)
+  - [Review Process](#review-process)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+Before submitting a bug report, please check if the issue has already been reported in the [issue tracker](https://github.com/nomtek/mistralai_client_dart/issues). If it hasn't, create a new issue with a clear title and description, including steps to reproduce the bug.
```

Comment:
It would be nice to also ask users to label the newly created issue with the `BUG` label 😉

Reply:
Updated in https://github.com/nomtek/mistralai_client_dart/pull/24/commits/42354570f194dd5e88f3982f0b824e9e1bbca23b